code/modules/surgery/organ_manipulation.dm

```diff
@@ -24,10 +24,12 @@
 /datum/surgery_step/manipulate_organs
 	time = 64
 	name = "manipulate organs"
-	implements = list(/obj/item/organ = 100, /obj/item/reagent_containers/food/snacks/organ = 0, /obj/item/organ_storage = 100)
+	implements = list(/obj/item/organ = 100, /obj/item/reagent_containers/food/snacks/organ = 0, /obj/item/organ_storage = 100, /obj/item/device/mmi = 100)
 	var/implements_extract = list(/obj/item/hemostat = 100, /obj/item/crowbar = 55)
 	var/implements_mend = list(/obj/item/cautery = 100, /obj/item/weldingtool = 70, /obj/item/lighter = 45, /obj/item/match = 20)
 	var/current_type
+	var/mends_the_incision = "mends the incision in"
+	var/mend_the_incision = "mend the incision in"//so we can reuse the whole thing for robotic surgery
```

Comment:
picky I know - can you make sure the order is the same in any instances of these. The order is flipped in `code/modules/surgery/organ_manip_robotic.dm` lines `32-33` for instance